outputs/policyadapter.go

```diff
@@ -57,4 +63,65 @@ func NewPolicyReportClient(config *types.Configuration, stats *types.Statistics,
 // PolicyReportPost creates Policy Report Resource in Kubernetes
 func (c *Client) PolicyReportCreate(falcopayload types.FalcoPayload) {
 	//to do
+	var crdclient *crdClient.Clientset
+	if c.Config.PolicyReport.Kubeconfig != "" {
+		restConfig, err := clientcmd.BuildConfigFromFlags("", c.Config.PolicyReport.Kubeconfig)
+		if err != nil {
+			fmt.Printf("unable to load kube config file: %v", err)
+		}
+		crdclient, err = crdClient.NewForConfig(restConfig)
+		if err != nil {
+			fmt.Printf("u %v", err)
+		}
+	}
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		fmt.Printf("unable to load in-cluster config: %v", err)
+	}
+
+	crdclient, err = crdClient.NewForConfig(restConfig)
+	if err != nil {
+		fmt.Printf("u %v", err)
+	}
+	ats := crdclient.Wgpolicyk8sV1alpha2().ClusterPolicyReports()
+	report := &clusterpolicyreport.ClusterPolicyReport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "dummy-policy-report",
+		},
+		Summary: v1alpha2.PolicyReportSummary{
+			//Fail: len(controls.Alert),
+		},
+	}
+	report.Results = append(report.Results, newResult(falcopayload))
+	result, err := ats.Create(context.TODO(), report, metav1.CreateOptions{})
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("Created policy-report %q.\n", result.GetObjectMeta().GetName())
```

Comment:
this doesn't respect the way to print logs

Reply:
Hey thomas, I will change this in my next commit.
 Thank you for bringing that to my notice.